iotdb/src/main/java/org/apache/iotdb/db/query/control/FileReaderManager.java

```diff
@@ -47,22 +47,26 @@
   private static final int MAX_CACHED_FILE_SIZE = 30000;
 
   /**
-   * the key of fileReaderMap is the file path and the value of fileReaderMap is the corresponding
+   * the key of closedFileReaderMap is the file path and the value of closedFileReaderMap is the corresponding
    * reader.
    */
-  private ConcurrentHashMap<String, TsFileSequenceReader> fileReaderMap;
+  private ConcurrentHashMap<String, TsFileSequenceReader> closedFileReaderMap;
+  private ConcurrentHashMap<String, TsFileSequenceReader> unclosedFileReaderMap;
```

Comment:
This variable `unclosedFileReaderMap` also need comment as `closedFileReaderMap`?

Reply:
added